text/profiles/0239-profiles-data-model.md

```diff
@@ -0,0 +1,1607 @@
+
+##### Field `original_payload`
+
+Original payload can be stored in this field. This can be useful for users who want to get the original payload.
+Formats such as JFR are highly extensible and can contain more information than what is defined in this spec.
+Inclusion of original payload should be configurable by the user. Default behavior should be to not include the original payload.
+If the original payload is in pprof format, it SHOULD not be included in this field.
+The field is optional, however if it is present `profile` MUST be present and contain the same profiling information.
+
+##### Field `profile`
+
+This is a reference to a pprof profile. Required, even when original_payload is present.
+</details>
+
+#### Message `Profile`
+
+Profile is a common stacktrace profile format.
+Measurements represented with this format should follow the
+following conventions:
+
+- Consumers should treat unset optional fields as if they had been
+  set with their default value.
+- When possible, measurements should be stored in "unsampled" form
+  that is most useful to humans.  There should be enough
+  information present to determine the original sampled values.
+- On-disk, the serialized proto must be gzip-compressed.
+- The profile is represented as a set of samples, where each sample
+  references a sequence of locations, and where each location belongs
+  to a mapping.
+- There is a N->1 relationship from sample.location_id entries to
+  locations. For every sample.location_id entry there must be a
+  unique Location with that id.
+- There is an optional N->1 relationship from locations to
+  mappings. For every nonzero Location.mapping_id there must be a
+  unique Mapping with that id.
+
+Represents a complete profile, including sample types, samples,
+mappings to binaries, locations, functions, string table, and additional metadata.
+
+<details>
+<summary>Field Descriptions</summary>
+
+##### Field `sample_type`
+
+A description of the samples associated with each Sample.value.
+For a cpu profile this might be:
+[["cpu","nanoseconds"]] or [["wall","seconds"]] or [["syscall","count"]]
+For a heap profile, this might be:
+[["allocations","count"], ["space","bytes"]],
+If one of the values represents the number of events represented
+by the sample, by convention it should be at index 0 and use
+sample_type.unit == "count".
+
+##### Field `sample`
+
+The set of samples recorded in this profile.
+
+##### Field `mapping`
+
+Mapping from address ranges to the image/binary/library mapped
+into that address range.  mapping[0] will be the main binary.
+
+##### Field `location`
+
+Locations referenced by samples via location_indices.
+
+##### Field `location_indices`
+
+Array of locations referenced by samples.
+
+##### Field `function`
+
+Functions referenced by locations.
+
+##### Field `attribute_table`
+
+A common table for strings referenced by various messages.
+string_table[0] must always be "".
```

Comment:
This might be related to a previous comment, but can you double-check this as well? If you do mean that the first entry on the table should be empty, the example should also be updated.

Reply:
fixed as it's the same kind of issue as [the previous one](https://github.com/open-telemetry/oteps/pull/239#discussion_r1463712873)